sycl/doc/UsersManual.md

```diff
@@ -257,6 +257,14 @@ and not recommended to use in production environment.
     options (e.g. -c, -E, -S) may interfere with the expected output set during
     the host compilation.  Doing so is considered undefined behavior.
 
+**`-fsycl-fp32-prec-sqrt`**
+
+    Enable use of correctly rounded `sycl::sqrt` function as defined by IEE754.
+    Without this flag, the default precision requirement for `sycl::sqrt` is 3
+    ULP.
+
+    NOTE: This flag is currently only supported with the CUDA and HIP targets.
```

Comment:
What happens if another target is used with this flag?

Reply:
It will just ignore the flag and print the following warning:
```
clang-13: warning: argument unused during compilation: '-fsycl-fp32-prec-sqrt' [-Wunused-command-line-argument]
```

As the argument isn't wired in the driver for the other targets.

Reply:
can we add a test to show this warning for other targets?

Reply:
Added a test to check the warning for the other targets

Reply:
Thanks